osler/core/utils.py

```diff
@@ -76,6 +76,15 @@ def return_duplicates(first_name_str, last_name_str):
         Q(last_name__in=last_name_var))
 
 
+def get_clindates(encounters):
+    '''For a qs of encounters, returns a list of unique clinic days'''
+    clindates = []
```

Comment:
(1) if you need to repeatedly test inclusion, you should use a set to get constant time performance for each operation. (2) is there a more Django-ey way to query for this?

Reply:
Anyway, I'm going to make this a potential new person project on the Notion and call it for now.